src/components/SearchCart/SearchCart.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const SearchCart = ({item}) => {
+  return (
+    <div>
+        <img src={`https://media.themoviedb.org/t/p/w188_and_h282_bestv2/${item.poster_path}`} alt={item.original_title} />
```

Comment:
let's move it to configs, so we can change this base url in one place

Reply:
@ArmenMesropyan done